dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -11,6 +11,27 @@ interface SiteSelectionForm {
   data?: SiteSelectionFormSchema;
 }
 
+const createValidationSchema = (
+  key: string,
+  formSchema: SiteSelectionFormSchema,
+) => {
+  let validationShape: any;
+
+  const property = formSchema.properties[key];
+
+  switch (property.type) {
+    case "string":
+      validationShape = string();
+      break;
+  }
+
+  if (validationShape && formSchema.required.includes(key)) {
+    validationShape = validationShape.required(`${property.title} is required`);
```

Comment:
I think we want to do something like this but I don't think `title` is the correct value to use. For the multiselect for example it would be "What use or uses do you think should be considered for this site? is required" which doesn't feel right.

For now lets just hardcode a value along the lines of "This is a required field"

Reply:
Done